flightmate/src/main/java/com/flightmate/dao/ApplicationDao.java

```diff
@@ -117,7 +117,7 @@ public void createPilotHoursTable() {
                          + "notes TEXT, "
                          + "approval_status VARCHAR(15) DEFAULT 'PENDING', "
                          + "created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP, "
-                         + "FOREIGN KEY (pilot_id) REFERENCES users(user_id));";
+                         + "FOREIGN KEY (pilot_id) REFERENCES users(user_id) ON DELETE CASCADE);";
```

Comment:
This would imply that if pilot hours get deleted, then the user gets deleted, no? I don't know if we want this behaviour.